lib/ansible/modules/utilities/logic/import_playbook.py

```diff
@@ -7,37 +7,39 @@
 __metaclass__ = type
 
 
-ANSIBLE_METADATA = {'metadata_version': '1.1',
-                    'status': ['preview'],
-                    'supported_by': 'core'}
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'core'
+}
 
 
 DOCUMENTATION = '''
 ---
-author:
-    - "Ansible Core Team (@ansible)"
+author: Ansible Core Team (@ansible)
 module: import_playbook
-short_description: import a playbook.
+short_description: Import a playbook
 description:
-     - Includes a file with a list of plays to be executed.
-     - Files with a list of plays can only be included at the top level, you cannot use this action inside a Play.
+  - Includes a file with a list of plays to be executed.
+  - Files with a list of plays can only be included at the top level, you cannot use this action inside a play.
 version_added: "2.4"
 options:
   free-form:
     description:
-        - This action allows you to specify the name of the file directly w/o any other options.
+      - The name of the imported playbook is specified directly without any other option.
 notes:
-    - This is really not a module, this is a feature of the Ansible Engine, as such it cannot be overridden the same way a module can.
+  - This is really not a module, this is a feature of the Ansible Engine, as such it cannot be overridden the same way
```

Comment:
Please revise to "This is a feature of the Ansible Engine, rather than a module, and cannot be overridden like a module."